Apps/ValidationTests/Win32/App.cpp

```diff
@@ -12,16 +12,21 @@
 
 #include <Shared/TestUtils.h>
 
+#include <Babylon/AppRuntime.h>
 #include <Babylon/Console.h>
-#include <Babylon/RuntimeWin32.h>
+#include <Babylon/NativeWindow.h>
+#include <Babylon/NativeEngine.h>
+#include <Babylon/ScriptLoader.h>
+#include <Babylon/XMLHttpRequest.h>
 
 #define MAX_LOADSTRING 100
 
     // Global Variables:
 HINSTANCE hInst;                                // current instance
 WCHAR szTitle[MAX_LOADSTRING];                  // The title bar text
 WCHAR szWindowClass[MAX_LOADSTRING];            // the main window class name
-std::unique_ptr<Babylon::RuntimeWin32> runtime{};
+std::unique_ptr<Babylon::AppRuntime> runtime{};
+std::string rootUrl{};
```

Comment:
why is this global now?

Reply:
Workaround for dangerous design decision that makes `XMLHttpRequest::Initialize` take a `const char*` that must outlive the `JsRuntime`. Touched on this in the comment I made in `.../Win32/App.cpp`. When that issue is fixed, this variable should also stop being global.